rerun_cpp/docs/readme_snippets.cpp

```diff
@@ -12,8 +12,6 @@ static std::vector<rerun::Color> create_colors() {
     return {};
 }
 
-// TODO(#3794): Once image logging is nicer, we should do that in this snippet as well!
```

Comment:
Not sure what this was referring to, there are no images in there :shrug:

Reply:
yeah that was the problem, at the time of writing we wanted to put images in there but they were awful to log. Now it's nice. Commited that in :)